wit-0.3.0-draft/timezone.wit

```diff
@@ -3,53 +3,33 @@ package wasi:clocks@0.3.0;
 @unstable(feature = clocks-timezone)
 interface timezone {
     @unstable(feature = clocks-timezone)
-    use wall-clock.{datetime};
+    use system-clock.{instant};
 
-    /// Return information needed to display the given `datetime`. This includes
-    /// the UTC offset, the time zone name, and a flag indicating whether
-    /// daylight saving time is active.
+    /// Return the IANA identifier of the currently configured timezone. The id
+    /// `UTC` indicates Coordinated Universal Time. Otherwise, this should be an
+    /// identifier from the IANA Time Zone Database.
     ///
-    /// If the timezone cannot be determined for the given `datetime`, return a
-    /// `timezone-display` for `UTC` with a `utc-offset` of 0 and no daylight
-    /// saving time.
-    @unstable(feature = clocks-timezone)
-    display: func(when: datetime) -> timezone-display;
-
-    /// The same as `display`, but only return the UTC offset.
+    /// For displaying to a user, the identifier should be converted into a
+    /// localized name by means of an internationalization API.
+    ///
+    /// In implementations that do not expose an actual time zone, this
+    /// should be the string `UTC`.
+    ///
+    /// In time zones that do not have an applicable name, a formatted
+    /// representation of the UTC offset may be returned, such as `-04:00`.
```

Comment:
Do we want to be precise about which formats are acceptable? I like [the Temporal grammar](https://tc39.es/proposal-temporal/#prod-UTCOffset) but it could be even more restrictive, e.g. only accept ±HH:MM.

Reply:
If we go the `to-debug-string` route, I'd say no.